feat: Add viewname on view generation

n0mad-d3v opened the pull request:
Hello Stacked Team,

The CLI is really great for very quickly creating all the screens of an application to start or make an MVP. 

Unfortunately, I find that there is a little something missing: display by default the name of the view that we have just generated: just with this small change, it makes things much clearer when we build the flow of the application and link all the screens without even having to add content everywhere!

What do you think ?

FilledStacks reviewed (approved):
Awesome

FilledStacks commented:
:tada: This PR is included in version 1.12.0 :tada:

The release is available on:
- `v1.12.0`
- [GitHub release](https://github.com/Stacked-Org/cli/releases/tag/v1.12.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

n0mad-d3v commented:
Hey @FilledStacks,
Thanks a lot for the merge! 😊

I just updated the CLI and tested and it doesn't seem to run. Should I have updated this file, too https://github.com/Stacked-Org/cli/blob/561e3a3a114ccaa41040cb95eb21f47275d1cb9d/lib/src/templates/compiled_templates.dart#L3267?

I was sure it'll be updated in the CI/CD but doesn't seem to be the case, my bad! 😥
Let me know how I could help to finish this feature, 

Thanks again!
